lib/conjur/api/audit.rb

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2013-2017 Conjur Inc
```

Comment:
2019?

Reply:
I was thinking about it - since it is a copied file template I didn't want to intervene with a future "replace all" action of all files or making this PR "dirty" with rebranding feature (Conjur ==> CyberArk Inc..) WDYT?

Reply:
A pattern we've been using with great success is to update the PR history prior to the merge to capture the distinct actions in the PR.  In this case, we could clean the history to include two commits:

> Adds support for Conjur's Audit Event endpoint.

> Project maintenance (which includes changing the copyright year and Postgres version)

This approach allows a reviewer to step through commits to understand scope.  We have a good overview here: https://github.com/conjurinc/docs/blob/master/reference/git.md#feature-branches

Reply:
OK, change only event.rb file